src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -35,6 +35,28 @@ public static string EmitHandlerDelegateType(this Endpoint endpoint)
         }
     }
 
+    public static string EmitHandlerDelegateCast(this Endpoint endpoint)
+    {
+        if (endpoint.Parameters.Length == 0)
+        {
+            return endpoint.Response.IsVoid ? "System.Action" : $"System.Func<{endpoint.Response.WrappedResponseType}>";
```

Comment:
Do these need to be `System.Action` and `System.Func`? Why not just `Action` and `Func`?

Reply:
Ah good call! We don't need them in this case because the cast gets emitted in the code block that already has access to the global usings.